traits/trait_types.py

```diff
@@ -314,19 +303,13 @@ def create_editor(self):
 
 
 class Complex(BaseComplex):
-    """ Defines a trait whose value must be a Python complex using a C-level
-        fast validator.
+    """ Defines a trait that holds a complex and uses a C-level fast validator.
```

Comment:
Urg. "a complex" is horrible. Replace with "a complex number"? Or "an instance of `complex`"?